RetailCoder.VBE/UI/Command/AddTestModuleCommand.cs

```diff
@@ -16,7 +15,7 @@ public class AddTestModuleCommand : CommandBase
         private readonly VBE _vbe;
         private readonly NewUnitTestModuleCommand _command;
 
-        public AddTestModuleCommand(VBE vbe, RubberduckParserState state, NewUnitTestModuleCommand command) : base(LogManager.GetCurrentClassLogger())
+        public AddTestModuleCommand(VBE vbe, NewUnitTestModuleCommand command) : base(LogManager.GetCurrentClassLogger())
```

Comment:
What's the difference between `AddTestModuleCommand`, and `NewUnitTestModuleCommand`? Shouldn't one *be* the other, and the other killed off?